zebrad/tests/acceptance.rs

```diff
@@ -71,8 +76,16 @@ fn help_no_args() -> Result<()> {
     let output = child.wait_with_output()?;
     let output = output.assert_success()?;
 
+    // First line haves the version
+    output.stdout_contains(r"zebrad [0-9].[0-9].[0-9]")?;
+
+    // Make sure we are in help by looking usage string
     output.stdout_contains(r"USAGE:")?;
 
+    // Make sure we have no info message in output
+    let notfound = output.stdout_contains(r"INFO");
+    assert!(notfound.is_err());
+
```

Comment:
I don't think there's a good way to check for any log message at any log level. And I don't think we want to match the exact usage message.

So let's delete this check.